docs/Salesforce-batchsink.md

```diff
@@ -81,18 +81,18 @@ Below is a non-comprehensive list of **sObjects** that are not currently availab
 
 **Operation:** Operation used for writing data into Salesforce.  
 Insert - adds records.  
-Upsert - upserts the records. Salesforce will decide if sObjects 
+Upsert - upserts the records. Salesforce will decide if sObjects
 are the same using external ID field.  
 Update - updates existing records based on Id field.
```

Comment:
Here and elsewhere in this file, ID should be in all caps (not "Id" or "id"), unless it's code in text.

Reply:
done